app/views/homepage/popular_links/_sidebar.html.erb

```diff
@@ -0,0 +1,9 @@
+<div class="sidebar__button sidebar__border">
+  <h2 class="sidebar__heading">Options</h2>
+  <% if @latest_popular_links.state == "published" %>
+    <%= primary_theme_navigation_button("Create new edition", :post, create_popular_links_path) %>
+  <% else %>
+    <%= primary_theme_navigation_button("Edit popular links", :get, edit_popular_links_path(@latest_popular_links)) %>
+    <%= secondary_theme_navigation_button("Publish", :post, publish_popular_links_path(@latest_popular_links)) %>
```

Comment:
Also might be better if we can use the appropriate variation of the button component here inside a form.

Reply:
same as previous, not sure if we can use Button component as we want to do a POST on one of the buttons, hence we tried to keep it consistent for both scenarios where we did a POST or GET.

Reply:
As Mark has said elsewhere that shouldn't be a problem and we really should be using Publishing Components at all times.